public/app/features/datasources/types.ts

```diff
@@ -16,3 +16,8 @@ export type DataSourcesRoutes = {
 };
 
 export type DataSourceTestStatus = 'success' | 'warning' | 'error';
+
+export type DataSourceInfo = {
+  dataSourcePluginName: string;
+  alertingSupported: boolean;
+};
```

Comment:
As it's only the `useDataSourceInfo()` hook using this type we could just move it in `useDataSourceInfo.tsx`?